package.json

```diff
@@ -1,12 +1,20 @@
 {
-  "name": "poe-wiki-search",
-  "version": "1.0.3",
-  "description": "Super simple search for poewiki.net!",
-  "main": "background.js",
-  "scripts": {},
-  "author": "github.com/pm5k",
-  "license": "GPL3",
-  "devDependencies": {
-    "eslint": "^7.32.0"
-  }
+    "name": "poe-wiki-search",
+    "version": "1.0.3",
+    "description": "Super simple search for poewiki.net!",
+    "main": "background/background.js",
+    "scripts": {
+        "start": "web-ext run -s ./src",
+        "test": "jest",
+        "test:watch": "jest --watchAll",
+        "build": "web-ext build -s ./src/ -a build --ignore-files **/*.spec.js"
+    },
+    "author": "github.com/pm5k",
```

Comment:
add a "contributors" key under here and add yourself into the array of contributors for this package. :)